GRID_MAP_WIDTH_IN_M: 20.0
GRID_MAP_HEIGHT_IN_M: 20.0
GRID_CELL_SIZE_IN_M: 0.1
MAP_BEHIND_VEHICLE_IN_M: 7
OCCUPIED: 1.0
FREE: 0.0
LIDAR_HEIGHT: 0.172
